controllers/backupcontroller.py

```diff
@@ -1,14 +1,4 @@
-from TestData import*
-from Controller import*
-class BackupController(Controller):
+class BackupController():
+  testData
```

Comment:
In Python, members are can only be declared from within methods. In this case, you're declaring testData in the constructor, so it will be visible from the point of the object's initialization with the self keyword. Line 2 will raise an error when Python attempts to compile the file.